chaos_kitten/brain/orchestrator.py

```
@@ -171,7 +173,7 @@ async def run_single_payload(payload_val, attack_conf, is_adaptive=False):
                     only_value if isinstance(only_value, str) else str(only_value)
                 )
             else:
-                payload_used = json.dumps(payload_val, sort_keys=True, default=str)
+                payload_used = json.dumps(payload_obj, sort_keys=True, default=str, ensure_ascii=True)
```

Ruff (0.15.1)

[error] Undefined name `payload_obj`

(F821)